ethminer/MinerAux.h

```diff
@@ -276,6 +276,19 @@ class MinerCLI
 					break;
 				}
 			}
+		else if(arg == "--cl-threads-per-hash" && i + 1 < argc) {
```

Comment:
CUDA calls it `--cuda-parallel-hash`...

Reply:
Yes, I originally thought these were different things, but it appears that you're right, and the intention for ```--cuda-parallel-hash``` is to set the number of threads per hash. I'll change it to ```--cl-parallel-hash``` and recommit. However, is this implemented for CUDA? I looked at the kernel and it seems like changing ```--cuda-parallel-hash``` doesn't affect the kernel

Reply:
its supposed to affect only cuda kernel, while using -G(OpenCL) cuda isn't being loaded so --cuda-parallel-hash will make no effect at all.